src/convertToObject.js

```diff
@@ -3,16 +3,33 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const lines = sourceString.split('\n');
+
+  lines.forEach(line => {
+    const match = line.match(/^\s*([\w-]+)\s*:\s*([^;]*)\s*;/);
+
+    if (match) {
+      const propertyName = match[1].trim();
```

Comment:
There is no real need to trim() the propertyName and propertyValue variables as the regex used in line 23 already excludes leading and trailing whitespaces.